sycl/source/detail/queue_impl.hpp

```diff
@@ -714,7 +714,7 @@ class queue_impl {
 
   const bool MHostQueue = false;
   // Assume OOO support by default.
```

Comment:
Could you please add a quick note here specifying what it means to have this be true?